easybuild/easyblocks/n/numpy.py

```diff
@@ -166,6 +166,21 @@ def get_libs_for_mkl(varname):
         if fft:
             extrasiteconfig += "\n[fftw]\nlibraries = %s" % fft
 
+        suitesparseroot = get_software_root('SuiteSparse')
+        if suitesparseroot:
+            amddir = os.path.join(suitesparseroot, 'AMD')
+            umfpackdir = os.path.join(suitesparseroot, 'UMFPACK')
```

Comment:
add the `os.path.exists` check I mentioned in a previous remark, just as a safe guard in case something changes in future `SuiteSparse` versions?